src/components/Button/src/Button.vue

```diff
@@ -104,6 +97,33 @@ function setColorVariables(tokens, variant) {
 	};
 }
 
+const TEXT_STYLES = new Set([
+	'fontFamily',
+	'fontWeight',
+	'fontSize',
+	'fontStyle',
+	'textTransform',
+	'textDecoration',
+]);
+
+function kebabCase(string) {
```

Comment:
we don't need to implement this, it already exists in [lodash](https://lodash.com/docs/4.17.15#kebabCase), and their impl works for camelCase identifiers that are longer than two words. let's remove this and add
```js
import { kebabCase } from 'lodash';
```
to the top of the file.